src/schemas/json/rust-project.json

```diff
@@ -0,0 +1,138 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "JSON schema for non-Cargo based Rust projects",
+    "type": "object",
+    "definitions": {
+        "crate": {
+            "type": "object",
+            "properties": {
+                "display_name": {
+                    "type": "string",
+                    "description": "Crate name used for display purposes, without affecting semantics."
```

Comment:
Optional fix (multiple issues): add link to official documentation with the following format: `"{{your_description}}\n{{link_here}}"`.
Note: without it I can't quickly check and suggest fixes for your schema.

Reply:
The only documentation for this file is [this one](https://rust-analyzer.github.io/manual.html#non-cargo-based-projects), which unfortunately doesn't have a lot of information about each property...